app/views/live/show_pages.html.erb

```diff
@@ -1,22 +1,26 @@
-<% set_page_title(@form.name) %>
+<% set_page_title(form.name) %>
 
-<% content_for :back_link, govuk_back_link_to(form_path(@form), t("back_link.form_view")) %>
+<% content_for :back_link, govuk_back_link_to(live_form_path(form), t("back_link.form_view")) %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-l govuk-!-margin-bottom-2">
-      <span class="govuk-caption-l"><%= @form.name %></span><span class="govuk-visually-hidden"> - </span>
+      <span class="govuk-caption-l"><%= form.name %></span><span class="govuk-visually-hidden"> - </span>
       <%= t("forms.form_overview.your_questions") %>
     </h1>
 
-    <%= render FormStatusTagDescriptionComponent::View.new(status: @form.status) %>
+    <%= render FormStatusTagDescriptionComponent::View.new(status: form.status) %>
 
     <h2 class="govuk-heading-m govuk-!-margin-top-7"></h2>
-    <% @pages.each_with_index do |page, index| %>
+    <% pages.each_with_index do |page, index| %>
       <%= render(SummaryCardComponent::View.new(
```

Comment:
My Summary Cards are coming out weird!

![image](https://user-images.githubusercontent.com/25597009/215807843-8397ac3e-7fd1-4469-bfeb-d278b8abb496.png)

Reply:
Sorted it, had some bad data in `public`